semgrep-core/src/tainting/Dataflow_tainting.ml

```diff
@@ -480,7 +562,11 @@ let rec check_tainted_lval env (lval : IL.lval) : Taints.t * Lval_env.t =
   let new_taints, lval_in_env, lval_env = check_tainted_lval_aux env lval in
   let taints_from_env = status_to_taints lval_in_env in
   let taints = Taints.union new_taints taints_from_env in
-  let sinks = lval_is_sink env.config lval |> Common.map trace_of_match in
+  let sinks =
+    lval_is_sink env.config lval
```

Comment:
I saw the `orig_is_sink` one first but I have a similar comment here too; I'm not familiar enough to know if the default should be `is_best_match`

Reply:
I did some refactoring but we still need the "raw" `orig_is_sink`.